(DOCSP-28135): Update event library details for new Scope object (#2605)
mongodb/docs-realm: source/sdk/swift/sync/event-library.txt

Events
~~~~~~

The ``AuditEvent`` collection where events are stored must have a schema 

> defined on the server for the App which you will be writing events to. 

Reviewer: Maybe?
```suggestion
defined on the server for the App receiving events. 
```
